README.md

```diff
@@ -22,6 +22,66 @@ Launch the server by typing:
 npm run dev
 ```
 
+## Local development
+
+### Ngrok setup
+
+Install ngrok globally:
+
+```bash
+npm install ngrok -g
+```
+
+Expose local server to public internet:
+
+```bash
+ngrok http 3000
+```
+
+If all goes well you should see `Tunnel States` is `online`, and Forwarding url.
+
+### Slack App setup
+
+Go to https://api.slack.com/apps?new_app=1 and create a new app.
+
+Copy Client ID, Client Secret and Signing Secret into `.env` at the root directory:
+
+```
+SLACK_CLIENT_ID=xxxxxxx
+SLACK_CLIENT_SECRET=xxxxxxx
+SLACK_CLIENT_SIGNING_SECRET=xxxxxxx
+```
+
+Click on the "Bot Users" tab, and click "Add a Bot User" button, specify a name for your bot, 
+and enable the option for "Always Show My Bot as Online", then click save.
+
+Click on the "OAuth & Permissions" tab in your Slack's app setting page, and under "Redirect URLs", add https://ngrok-url/oauth, then click save.
+
+Click on the "Interactive Components" tab, under "Request URL", add https://ngrok-url/slack/receive, then click save.
+
+Click on the "Event Subscriptions" tab, and switch on "Enable Events", under "Request URL", add https://ngrok-url/slack/receive. 
+Once finish typing, Slack will verify that this endpoint is properly configured, the local must be running and exposed to public internet to make this work.
```

Comment:
"Once you're finished typing," 
Instead of "the local must be running", "you must have your localhost running" might be clearer